src/components/SearchModal/CommonBases.tsx

```diff
@@ -79,6 +79,7 @@ export default function CommonBases({
           const { symbol } = getDisplayTokenInfo(token)
           return (
             <BaseWrapper
+              className="favorite-token"
```

Comment:
I think it'd be better using `id={}` or `data-id={}`. `id` and `data-id` mean identifiers. Class names should be used for styling only

Reply:
I've updated. Please review again.